### DISCUSSION TITLE
Enable Timebase metrics

### DISCUSSION THREAD

**Discussion by ShivaniSK** [finos/TimeBase-CE#274] [Q&A]
Hi @alex-karpovich 
We have installed timebase using "Command Line Installer".
We have enabled the timebase metrics and aggregator metrics as can be seen in the screen snip below

![image](https://github.com/finos/TimeBase-CE/assets/8392899/2c02f433-dd38-4157-a642-c7e6a3929275)

We referred below timebase docs
https://kb.timebase.info/docs/development/sys-metrics#adminproperties 
https://kb.timebase.info/community/development/aggregator/aggregator-monitoring#step-1-enable-metrics

However, we are unable to access the below urls.
```
timebase host:8011/tb/metrics
timebase host:8011/agg/api/metrics
```

It gives 404 not found error.
Can you please let us know if we are missing out on any changes.

Regards,
Shivani

Regards,
Shivani

**Comment by ShivaniSK** [comment]
Hello @alex-karpovich ,
Can you please have a look into the above issue. We have updated the description based on the issue we are facing.

**Reply by alex-karpovich** [reply]
Metrics is not enabled with CE version - going to add them on this week

**Reply by ShivaniSK** [reply]
Thanks @alex-karpovich , do let us know once the metrics are enabled.
